packages/suite/src/services/suite/invityAPI.ts

```diff
@@ -0,0 +1,295 @@
+import { createHash } from 'crypto';
+import {
+    ExchangeListResponse,
+    ExchangeTradeQuoteResponse,
+    ExchangeTradeQuoteRequest,
+    ConfirmExchangeTradeRequest,
+    ExchangeTrade,
+    WatchExchangeTradeResponse,
+    ExchangeCoinInfo,
+    BuyListResponse,
+    BuyTradeQuoteRequest,
+    BuyTradeQuoteResponse,
+    BuyTradeRequest,
+    BuyTradeResponse,
+    BuyTradeFormResponse,
+    BuyTrade,
+    WatchBuyTradeResponse,
+    CountryInfo,
+    SupportTicketResponse,
+    SupportTicket,
+} from 'invity-api';
+import { isDev } from '@suite-utils/build';
+
+// TODO - get from constants
+const BitcoinTestnetTicker = 'TEST';
+const EthereumTestnetRopstenTicker = 'tROP';
```

Comment:
shouldn't be one for tXRP (ripple testnet) as well?

Reply:
I removed all testnet coins, they are not necessary. The code was leftover from another Invity project.